docs/howto/openstack/neutron/new-network-control-panel.md

```diff
@@ -1,57 +1,332 @@
+one network to make the new server a member of. Since you may have
+more than one network per region, let us now walk through creating two
+new networks; one using the {{extra.gui}}, and another using the
+OpenStack CLI.
 
-On the top right-hand side of the {{extra.gui}}, click the _Create_
-button. A new pane will slide into view from the right-hand side of
-the browser window, titled _Create_.
+## Prerequisites
 
-![Create a new object](assets/new-net-panel/shot-01.png)
+Whether you choose to work from the {{extra.gui}} or with the
+OpenStack CLI, you need to [have an
+account](/howto/getting-started/create-account) in {{extra.brand}}
+Cloud. Additionally, to use the OpenStack CLI make sure to [enable it
+first](/howto/getting-started/enable-openstack-cli).
 
-You will notice several rounded boxes prominently displayed on that 
-pane, each for defining, configuring, and instantiating a different 
-{{extra.brand}} Cloud object. Go ahead and click the _Network_ box. A 
-new pane titled _Create Network_ will slide over. At the top, type in a 
-name and select one of the available regions for the new network.
+## Creating a network
 
-![New network name and region](assets/new-net-panel/shot-02.png)
+To create a network from the {{extra.gui}}, fire up your favorite web
+browser, navigate to the [Cleura Cloud](https://{{extra.gui_domain}})
+page, and login into your {{extra.brand}} account. On the other hand,
+if you prefer to work with OpenStack CLI, please do not forget to
+source the relative RC file first.
 
-Expand the _Advanced Options_ section below, make sure _Port 
-Security_ is enabled, and leave the MTU parameter blank.
+=== "{{extra.gui}}"
+    On the top right-hand side of the {{extra.gui}}, click the _Create_
+    button. A new pane will slide into view from the right-hand side of
+    the browser window, titled _Create_.
 
-![MTU and port security](assets/new-net-panel/shot-03.png)
+    ![Create a new object](assets/new-net/shot-01.png)
 
-You probably want a full-featured network for your cloud servers, so
-please activate the _Create a complete network containing a subnet and a router_
-option. You will notice that
-[a network address in CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation)
-is pre-configured for your network. You also get a couple of
-DNS servers, a Gateway, and a DHCP server.
+    You will notice several rounded boxes prominently displayed on that
+    pane, each for defining, configuring, and instantiating a different
+    {{extra.brand}} Cloud object. Go ahead and click the _Network_ box. A
+    new pane titled _Create Network_ will slide over. At the top, type
+    in a name and select one of the available regions for the new
+    network.
 
-![Complete network](assets/new-net-panel/shot-04.png)
+    ![New network name and region](assets/new-net/shot-02.png)
+=== "OpenStack CLI"
+    Start by creating a new network, named `nuremberg`:
```

Comment:
Please don't call the network `nuremberg`. People might read this as a [dog whistle](https://en.wikipedia.org/wiki/Dog_whistle_(politics)) reference to Nazi party rallies.

The GUI examples use the innocuous name `nordostbahnhof` for the network, so in the interest of parallelism, use that here, too.

Reply:
My idea is to create two different networks, each using a different method. (I mention this in the opening paragraph, by the way.) I understand how `nuremberg` can be misunderstood, though. So, my suggestion is to do a `s/nuremberg/hauptbahnhof/g`. What do you think?

Reply:
No. You are documenting, in one page, how to create a network with the GUI, and with the CLI. Please don't switch up the names. Use one network name throughout.